### PR TITLE
adding DevSecOps WoW and issue template

### PR DESCRIPTION
**Pull request by devopsmatt**
This PR adds proposed DevSecOps WoW document and a GH template for DevSecOps work issues.
Please review and comment - we want all team leads to be comfortable with the final version of the WoW, before we merge it and officially adopt.

### CODE REVIEW

File: DevSecOps-WoW.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# CDC ReportStream DevSecOps Ways of Working
```
Comment (arnejduranovic) [inline DevSecOps-WoW.md:1]: I think this document would be more appropriate in the team sharepoint? If you agree, I can move it there once everyone has approved the PR.

File: DevSecOps-WoW.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- The DevSecOps Team will track all DevSecOps issues on the ReportStream board that are tagged with the `DevSecOps` and `reportstream` labels.
+- When new issues are created using the DevSecOps issue template, the DevSecOps Team will receive an automatic Slack notification in a separate DevSecOps alert channel (TBD).
+
+## Workflow Management/Task Prioritization
+
+- The DevSecOps Team will operate using a Kanban approach and stay within established Work in Progress (WIP) limits.
+- While the DevSecOps Team won't be embedded with specific Apps Teams, they will still operate within the same sprint cadence as the Apps Teams.
+- DevSecOps will review new tickets as they come in and prioritize them.
+- Every 2 weeks on Wednesday, a DevSecOps Team representative will attend a Scrum of Scrums with the Apps Tech Leads (and TBD) to review the status of in progress items, flag blockers, and discuss prioritization of new issues.  Patrick will give ultimate guidance on what issues need to take priority over others in scenarios, wherever it is unclear.
```
Comment (arnejduranovic) [inline DevSecOps-WoW.md:21]:
```suggestion
- Every 2 weeks on Wednesday, a DevSecOps Team representative will attend a Scrum of Scrums with the Apps Tech Leads (and TBD) to review the status of in progress items, flag blockers, and discuss prioritization of new issues. The project engineering lead will give ultimate guidance on what issues need to take priority over others in scenarios, wherever it is unclear.
```

File: DevSecOps-WoW.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- DevSecOps Team members are not required to attend Apps Team stand ups, sprint planning, refinement, or retro meetings given that they may be doing work across multiple apps at a given time. That said, they will join Apps Team stand ups at their discretion if they need to communicate with the Team, and/or seek clarifications or are requested to join to provide input. They will also pair with Tech Leads to flesh out draft issues, when their input is needed.
+- (Eventually) DevSecOps Team members will not factor into app Team velocity calculations.
```
Comment (arnejduranovic) [inline DevSecOps-WoW.md:23]:
```suggestion
- DevSecOps Team members will not factor into app Team velocity calculations.
```

File: DevSecOps-WoW.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- The DevSecOps Team will conduct their own internal stand ups for 10-15 minutes and other recurring Agile ceremonies.
+- DevSecOps Team members will communicate status of their tickets with the respective App Tech Leads.
```
Comment (arnejduranovic) [inline DevSecOps-WoW.md:25]: New item proposal at the bottom:

- DevSec0ps team shall participate in, and help inform, quarterly program planning exercises

cc @chris-kuryak

**Review by arnejduranovic — APPROVED** [review]

File: DevSecOps-WoW.md (hunk 1 above)
Comment (devopsmatt) [reply DevSecOps-WoW.md:1]: Sure, we can move it once we have a quorum.  I don't have access to SP yet, so I pushed here to open the WoW to discussion to erasure everyone is comfortable with what I'm proposing on behalf of my team.

File: DevSecOps-WoW.md (hunk 4 above)
Comment (devopsmatt) [reply DevSecOps-WoW.md:25]: Agreed.  Adding.

File: DevSecOps-WoW.md (hunk 2 above)
Comment (devopsmatt) [reply DevSecOps-WoW.md:21]: Agreeed.  Committing the suggestion..

File: DevSecOps-WoW.md (hunk 1 above)
Comment (devopsmatt) [reply DevSecOps-WoW.md:1]: The WoW has relative hrefs to the issue template, it'll need to be updated so the links work from SP.

**Review by victor-chaparro — APPROVED** [review]

**Review by MichaelEsuruoso — APPROVED** [review]
1.will team be informed on the WIP limits and if devops are at the limit does that stop us from requesting support until your WIP is cleared?

2.how will task be prioritized across teams ? examples if the platform team and engagement team botht submit a ticket and both are considered blockers how does prioritization factor in the request .

File: DevSecOps-WoW.md (hunk 1 above)
Comment (devopsmatt) [reply DevSecOps-WoW.md:1]: Hi @MichaelEsuruoso 
1. Our WiP limit will be what we set it to.  Before we can set it, we need to run a few sprints to get realistic metrics.  We can always stay within the WiP limits as we can deprioritize work that's being replaced with higher priority item.   

2.  That's where the project engineering lead steps in and helps establish priorities.  We can work on multiple items concurrently.

**Review by pstonebu — APPROVED** [review]
This all looks great to me.

**Review by emvaldes — COMMENTED** [review]
I am in favor of the improvements and changes made to this document.

**Review by emvaldes — APPROVED** [review]
Approving changes from the DevSecOps team.

**Pull request merged**